### ISSUE TITLE
Issue about ScanNet file

### ISSUE THREAD

**Issue by hugoycj** [xheon/JointEmbedding#4]
Thank you for your great world.
I wonder if you could give us some instructions about preprocess scannet file. For example, the input format of scannet is '.mask', what's the content of it. 
Great thanks for your help

**Comment by xheon** [comment]
Hi, thanks for your interest in the project.
I'm a bit busy with teaching at the moment, but I will come back to you next week. Feel free to ping again!

**Comment by shingkim** [comment]
> Thank you for your great world. I wonder if you could give us some instructions about preprocess scannet file. For example, the input format of scannet is '.mask', what's the content of it. Great thanks for your help

hi, I also get the same problem. Have you solved it？thanks

**Comment by xheon** [comment]
Hi, sorry for the late reply.

You can find the preprocessed data for both scannet and shapenet together with pretrained weighted here: https://hidrive.ionos.com/share/fg649wmiu5

Please note, that the pre-trained weights refer to the code before this commit (.e. without the 2 sigmoid before the separation model.): https://github.com/xheon/JointEmbedding/commit/4ecbb03d0f0944093b4526077a2df3db61f3b54b

`python test.py --gpu 0 --scannet_path scannet_data_path --shapenet_path shapenet_data_path --confusion_val_path one_to_one_mapping_validation.json --confusion_train_path one_to_one_mapping_train.json --confusion_test_path one_to_one_mapping_test.json --similarity_path scan-cad_similarity_public_augmented-100_v0.json --model_path joint-embedding`

retrieval accuracy:

```
correct: 2144, total: 5102, accuracy: 0.42022736181889453
02747177:   183/  562 --> 0.326
02808440:    35/   74 --> 0.473
02818832:    91/  165 --> 0.552
02871439:    94/  369 --> 0.255
02933112:    86/  425 --> 0.202
03001627:  1026/ 1828 --> 0.561
03211117:    76/  208 --> 0.365
03337140:    27/  106 --> 0.255
04256520:   100/  265 --> 0.377
04379243:   313/  771 --> 0.406
other:   113/  329 --> 0.343
```
ranking quality

```
correct: 2104, total: 13110, accuracy: 0.1604881769641495
02747177:   141/ 1152 --> 0.122
02808440:    40/  169 --> 0.237
02818832:    81/  429 --> 0.189
02871439:    93/  867 --> 0.107
02933112:    97/ 1010 --> 0.096
03001627:  1016/ 5016 --> 0.203
03211117:    84/  590 --> 0.142
03337140:    25/  251 --> 0.100
04256520:   107/  753 --> 0.142
04379243:   321/ 2099 --> 0.153
other:    99/  774 --> 0.128
```
Best,
Manuel

**Comment by shingkim** [comment]
thanks




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Manuel ***@***.***&gt;; 
发送时间: 2022年4月26日(星期二) 上午7:15
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [xheon/JointEmbedding] Issue about ScanNet file (#4)





 
关闭#4。
 
—
直接回复此邮件，在 GitHub 上查看，或取消订阅。
您收到此消息是因为您发表了评论。消息 ID：<xheon/JointEmbedding/issue/4/issue_event/6494165097 @ github 。com&gt;
